src/OutgoingMessage.ts

```diff
@@ -40,27 +41,29 @@ export default class OutgoingMessage extends NativeOutgoingMessage {
 
   // Those methods cannot be prototyped because express explicitelly overrides __proto__
   // See https://github.com/expressjs/express/blob/master/lib/middleware/init.js#L29
-  public end: NativeOutgoingMessage["end"] = (
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    chunkOrCb?: any
+  public readonly end: NativeOutgoingMessage["end"] = (
+    chunkOrCb: Parameters<NativeOutgoingMessage["end"]>[0]
   ) => {
     // 1. Write head
+    // eslint-disable-next-line no-invalid-this
```

Comment:
`no-invalid-this` is triggered because `end` is defined as a property rather than a class method (which may lead to  context override, so it could be unsafe to use this here).

As we are kinda _hacking_ Express' APIs, I'd rather keep the code as is.

Reply:
ok